api/app/signals/apps/history/models/log.py

```diff
@@ -64,17 +65,24 @@ def __str__(self):
     #
     # To keep the history endpoint intact the following functions are defined to mimic the history view in the database
 
-    @staticmethod
-    def translate_content_type(content_type_name):
+    def translate_content_type(self, content_type_name):
         """
         Present for backwards compatibility
         """
-        content_type_translations = {'category assignment': 'category_assignment',
-                                     'service level objective': 'sla',
-                                     'type': 'type_assignment',
-                                     'signal user': 'user_assignment'}
+        content_type_translations = {
+            'category assignment': 'category_assignment',
+            'service level objective': 'sla',
+            'type': 'type_assignment',
+            'signal user': 'user_assignment',
+        }
+
         if content_type_name in content_type_translations:
             return content_type_translations[content_type_name]
+        elif content_type_name == 'signal departments':
```

Comment:
Damn! In Amsterdam we completely overlooked this. I can seen the "Actie onbekend" when the "Verantwoordelijke afdeling" is set in the history of a Signal.